internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider_test.go

```diff
@@ -92,17 +104,17 @@ func importStateIDFunc(federationSettingsID, idpID string) resource.ImportStateI
 	}
 }
 
-func configBasic(federationSettingsID, ssoURL, issuerURI string) string {
+func configBasic(federationSettingsID, ssoURL, issuerURI, associatedDomain string) string {
 	return fmt.Sprintf(`
 	resource "mongodbatlas_federated_settings_identity_provider" "test" {
-		federation_settings_id = "%[1]s"
-		name = "mongodb_federation_test"
-        associated_domains           = ["reorganizeyourworld.com"]
-        sso_debug_enabled = true
-        status = "ACTIVE"
-        sso_url = "%[2]s"
-        issuer_uri = "%[3]s"
-        request_binding = "HTTP-POST"
+		federation_settings_id 		= %[1]q
```

Comment:
nit: can we fix the format? Usually when this happens is because of a mix of spaces and tabs

Reply:
Thanks. Added in: [0bdf6d5](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2290/commits/0bdf6d5fda97b26440bcc9fb8c6d58cdf0d15bd0)